src/scripts/countdown.js

```diff
@@ -0,0 +1,12 @@
+const getTimeFormate = require('./timeFormate');
+
+module.exports = () => {
+  const times = Array.from(document.getElementsByClassName('countdown'));
+  setInterval(() => {
+    times.forEach(elem => {
+      const time = elem.getAttribute('data-src');
+      const timeDiff = getTimeFormate(time - Date.now());
+      elem.innerHTML = `${timeDiff} (${getTimeFormate(time)} Uhr)`;
+    });
+  }, 100);
```

Comment:
Do we really need to check 10 times a second or would it be sufficient to run this once or twice a second?